app/cli/cmd/integration_add_deptrack.go

```diff
@@ -29,17 +29,24 @@ func newIntegrationAddDepTrackCmd() *cobra.Command {
 	var allowAutoCreate bool
 
 	cmd := &cobra.Command{
-		Use:     "dependency-track",
-		Aliases: []string{"deptrack"},
-		Short:   "Add Dependency-Track integration ",
+		Use:        "dependency-track",
+		Aliases:    []string{"deptrack"},
+		Short:      "Add Dependency-Track integration ",
+		Deprecated: "use `chainloop integration add` instead",
```

Comment:
I am confused, so we are deprecating the old method with "dependency-track" and the new method will follow almost the same UX but the key is "dependencytrack"?

Reply:
> and the new method will follow almost the same UX but the key is "dependencytrack"?

The new command will use as first argument whatever `id` the extension you are invoking has. In our case `dependencytrack` yes. 

What is the source of the confusion? The deprecation? The fact that we are building a new functionality on a parent command?

Reply:
> I would suggest to exit with error and a proper deprecation message in case of dependency-track.

but that will break the current commands in the guides no?

And there is a deprecation message already


```
$ go run main.go --insecure integration add dependency-track 
Command "dependency-track" is deprecated, use `chainloop integration add` instead
```

Reply:
>  Now there are two working versions: dependency-track and dependencytrack. Are we going to maintain both?

my idea was to first, deprecate the old command (as I did) and once we stabilize the new registration/attachment methods and update the docs we can fully remove it. WDYT?